File: admin/src/api/interface/systemFile.ts
// system_file 文件管理
import { ReqPage } from "./index";
export namespace SystemFile {
  export interface ReqSystemFileList extends ReqPage {
    deleted?: number; // 是否删除
    fileType?: number; // 文件类型
    fileName?: string; // 文件名称
    deptId?: number; // 部门
  }
  export interface ResSystemFileItem {
    id: number; // 编号
    fileName: string; // 文件名称
    fileType: number; // 文件类型
    fileMimeType: string; // Mime类型
    fileSize: number; // 文件大小
    filePath: string; // 文件路径
    userId: number; // 用户 ID
    deleted: number; // 是否删除
    tenantId: number; // 租户
    creator: string | undefined; // 创建人
    createTime: string | undefined; // 创建时间
    updater: string | undefined; // 更新人
    updateTime: string | undefined; // 更新时间
  }
}

File: cloud/dao/system_file.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// SystemFile 文件管理 system_file
type SystemFile struct {
	Id           *int64        `db:"id" json:"id"`                       //bigint 编号,PRI
	FileName     *string       `db:"file_name" json:"fileName"`          //varchar 文件名称
	FileType     *int32        `db:"file_type" json:"fileType"`          //tinyint 文件类型
	FileMimeType *string       `db:"file_mime_type" json:"fileMimeType"` //varchar Mime类型
	FileSize     *int64        `db:"file_size" json:"fileSize"`          //bigint 文件大小
	FilePath     *string       `db:"file_path" json:"filePath"`          //varchar 文件路径
	TenantId     *int64        `db:"tenant_id" json:"tenantId"`          //bigint 租户
	Creator      null.String   `db:"creator" json:"creator"`             //varchar 创建人
	CreateTime   null.DateTime `db:"create_time" json:"createTime"`      //datetime 创建时间
	Updater      null.String   `db:"updater" json:"updater"`             //varchar 更新人
	UpdateTime   null.DateTime `db:"update_time" json:"updateTime"`      //datetime 更新时间
}
